clients/admin-ui/src/features/config-wizard/OrganizationInfoForm.tsx

```diff
@@ -31,7 +31,7 @@ const useOrganizationInfoForm = (handleChangeStep: Function) => {
       const organizationBody = {
         name: values.name,
         description: values.description,
-        fides_key: "default_organization",
+        fides_key: "default_organization_21",
```

Comment:
what is the `21`?

Reply:
Oh, that's not supposed to be there, fixing now 😆